packages/terra-application-navigation/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ Changelog
 
 Unreleased
 ----------
+### Fixed
+* Focus should be returned to latest selected key when navigation prompt is dismissed.
+* Focus should be passed to `Content` when navigation prompt is accepted.
```

Comment:
Update here :- Navigation prompt is accepted and Tab key is pressed

Reply:
Updated here https://github.com/cerner/terra-framework/pull/1124/commits/a5857599a27987dc48f9fa5061aebbdc3d8d658d.